4_switch_controlled_motor_normal/main/switch_control_motor_normal.c

```diff
@@ -19,62 +19,70 @@ void drive_task(void *arg)
 
-	//infinite loop to get LSA readings continuously
+	//infinite loop 
 	while (1)
 	{
-		// check input from LSA and activate the appropriate motor accordingly.
+		// check which switch is activated and run appropriate motor accordingly.
 
 		/**
-		 * read_switch() checks if LSA has given any input to the switch and if statement is true if input is given
+		 * read_switch() checks if the given switch is activated or not
 		 * more information can be found at https://github.com/SRA-VJTI/sra-board-component/blob/9f28700759ac816660c18859d65303d6540e8732/src/switches.c#L36-L46
 		 */
 		if (read_switch(SWITCH_1))
 		{
 			/**
-			 * runs the first motor A_0 for 80 duty cylces forward or backward according to LSA input
+			 * runs the first motor A_0 for 80 duty cycles forwards 
 			 * more details can be found at https://github.com/SRA-VJTI/sra-board-component/blob/9f28700759ac816660c18859d65303d6540e8732/src/motor_driver.c#L195
 			 */
 			set_motor_speed(MOTOR_A_0, MOTOR_FORWARD, 80);
 
-			// Display logs - final LSA readings
+			// Display logs - current status of MOTOR_A_0
 			ESP_LOGI(TAG_SWITCH_CONTROL_MOTOR, "MOTOR_A_0 FORWARD");
 		}
 		else if (read_switch(SWITCH_2))
 		{
+			/**
+			 * runs the first motor A_0 for 80 duty cycles backwards
+			 * more details can be found at https://github.com/SRA-VJTI/sra-board-component/blob/9f28700759ac816660c18859d65303d6540e8732/src/motor_driver.c#L195
+			 */
 			set_motor_speed(MOTOR_A_0, MOTOR_BACKWARD, 80);
 			ESP_LOGI(TAG_SWITCH_CONTROL_MOTOR, "MOTOR_A_0 BACKWARD");
 		}
 		else
 		{
-			// if there is no input from  LSA for first motor then stop the motor, this stops the motor if the previous 80 duty cycles are not completed
+			// if neither of switch 1 or switch 2 is pressed on then stop the motor A_0
 			set_motor_speed(MOTOR_A_0, MOTOR_STOP, 0);
 		}
 
 		if (read_switch(SWITCH_3))
 		{
 			/**
-			 * runs the second motor A_1 for 80 duty cylces forward or backward according to LSA input
+			 * runs the second motor A_1 for 80 duty cycles forwards
 			 * more details can be found at https://github.com/SRA-VJTI/sra-board-component/blob/9f28700759ac816660c18859d65303d6540e8732/src/motor_driver.c#L195
 			 */
 			set_motor_speed(MOTOR_A_1, MOTOR_FORWARD, 80);
 
-			//Displaying Logs - final LSA readings
+			// Display logs - current status of MOTOR_A_1
 			ESP_LOGI(TAG_SWITCH_CONTROL_MOTOR, "MOTOR_A_1 FORWARD");
 		}
 		else if (read_switch(SWITCH_4))
 		{
+			/**
+			 * runs the second motor A_1 for 80 duty cycles backwards 
+			 * more details can be found at https://github.com/SRA-VJTI/sra-board-component/blob/9f28700759ac816660c18859d65303d6540e8732/src/motor_driver.c#L195
+			 */
 			set_motor_speed(MOTOR_A_1, MOTOR_BACKWARD, 80);
 			ESP_LOGI(TAG_SWITCH_CONTROL_MOTOR, "MOTOR_A_1 BACKWARD");
 		}
 		else
 		{
-			//if there is no input from LSA for the second motor then stop the motor
+			// if neither of switch 3 or switch 4 is pressed on then stop the motor A_1
```

Comment:
Change to "  if neither of switch 3 or switch 4 is pressed, stop the motor A_1. "